### PR TITLE
Introduce mobile tests screenshots

### PR DESCRIPTION
**Pull request by KKoukiou**

### CODE REVIEW

**Comment by KKoukiou** [comment]
Doh TEST_MOBILE does not get picked up here.  Need to recheck this.

**Comment by martinpitt** [comment]
> Doh TEST_MOBILE does not get picked up here

Need to bump test/common checkout?

**Comment by KKoukiou** [comment]
> > Doh TEST_MOBILE does not get picked up here
> 
> Need to bump test/common checkout?

I have already done this. It seems that the exported TEST_MOBILE option is not set any more where it's being read.

File: test/run [hunk 1]
```diff
@@ -7,3 +7,5 @@ set -eu
 make codecheck
 make check
 make po/machines.pot
+
+[ "${TEST_SCENARIO}" = "${TEST_SCENARIO%%mobile}" ] || export TEST_MOBILE=true
```
Comment (KKoukiou) [inline test/run:11]: Lol I just need to add this above the make check :P

**Review by martinpitt — APPROVED** [review]
Nice! Amazing that none of the tests need a mobile special-case.

I re-triggered the stuck tests.

**Comment by martinpitt** [comment]
There is something fishy here, srpm build in packit [keeps failing](https://dashboard.packit.dev/results/srpm-builds/34763) -- it does not find a pre-built webpack. I think this is due to packit now trying to rebase an outdated branch? I think this is reasonably safe to ignore.

**Pull request merged**